src/20_Components/amp-form.html

```diff
@@ -0,0 +1,79 @@
+<!---{
+    "experiment": true,
+    "component": "amp-form",
+    "draft": true
+  }--->
+<!--
+  #### Introduction
+
+  The [amp-form](https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/amp-form.md0) extension allows the usage of forms and input fields in an AMP document.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Import the library.
+  -->
+  <script async custom-element="amp-form" src="https://cdn.ampproject.org/v0/amp-form-0.1.js"></script>
+  <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script> 
+  <!-- 
+    Customize success or error messages by using `amp-form-submit-success` and `amp-form-submit-error` classes.
+   -->
+  <style amp-custom>
+        form.amp-form-submit-success [submit-success],  
+          form.amp-form-submit-error [submit-error]{
+            margin-top: 4px;
+            font-size: 20px;
+        }
+        form.amp-form-submit-success [submit-success] {
+            color: #ff4081;
+        }
+        form.amp-form-submit-error [submit-error] {
+            color: #dc4e41;
+        }
+        .button-submit {
+          margin-top: 4px;
+        }
+    </style>
+</head>
+<body> 
+
+  <!-- #### Non XHR Request -->
```

Comment:
I don't find "Non XHR Request" a good title - it doesn't describe what this approach is about.